Rocket.Chat/Controllers/WebViewEmbedded/WebViewControllerEmbedded.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  WebViewControllerEmbedded.swift
+//  Rocket.Chat
+//
+//  Created by Rafael Kellermann Streit on 11/24/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+class WebViewControllerEmbedded: UIViewController {
```

Comment:
Can you add an ActivityIndicator like we do in OAuthViewController.swift? It stays white for a long time here.